src/icons/sponsors/woollymammoth.svg

```diff
@@ -0,0 +1,10 @@
+<path d="M286.593 58.8397H279.023V19.7465H285.788L286.513 24.863C287.533 22.9443 289.063 21.4253 291.103 20.3061C293.197 19.1869 295.586 18.6272 298.27 18.6272C304.39 18.6272 308.551 21.2654 310.752 26.5418C311.665 24.0902 313.249 22.1715 315.504 20.7858C317.759 19.3467 320.416 18.6272 323.476 18.6272C327.986 18.6272 331.529 19.933 334.106 22.5446C336.683 25.1561 337.972 28.727 337.972 33.2572V58.8397H330.563V35.096C330.563 32.058 329.758 29.6863 328.147 27.9808C326.59 26.2753 324.47 25.4226 321.785 25.4226C318.94 25.4226 316.631 26.3553 314.859 28.2207C313.141 30.0861 312.282 32.5644 312.282 35.6556V58.8397H304.793V35.016C304.793 31.9781 304.015 29.633 302.458 27.9808C300.901 26.3286 298.78 25.5025 296.096 25.5025C293.197 25.5025 290.888 26.4352 289.17 28.3006C287.452 30.1127 286.593 32.5644 286.593 35.6556V58.8397Z" fill="white"/>
+<path d="M349.271 54.043C345.405 50.2056 343.473 45.2757 343.473 39.2531C343.473 33.2306 345.405 28.3006 349.271 24.4632C353.136 20.6259 358.049 18.7072 364.008 18.7072C369.914 18.7072 374.799 20.6259 378.665 24.4632C382.53 28.3006 384.463 33.2306 384.463 39.2531C384.463 45.2757 382.53 50.2056 378.665 54.043C374.799 57.8804 369.914 59.799 364.008 59.799C358.049 59.799 353.136 57.8804 349.271 54.043ZM354.666 29.26C352.304 31.8182 351.123 35.1493 351.123 39.2531C351.123 43.357 352.304 46.688 354.666 49.2463C357.082 51.8045 360.196 53.0836 364.008 53.0836C367.82 53.0836 370.907 51.8045 373.269 49.2463C375.685 46.6347 376.893 43.3037 376.893 39.2531C376.893 35.2026 375.685 31.8982 373.269 29.3399C370.907 26.7284 367.82 25.4226 364.008 25.4226C360.196 25.4226 357.082 26.7017 354.666 29.26Z" fill="white"/>
+<path d="M401.147 58.8397H393.577V26.0622H385.846V19.7465H393.577V7.51486H401.147V19.7465H408.878V26.0622H401.147V58.8397Z" fill="white"/>
+<path d="M420.825 58.7598H413.255V0H420.825V25.7424C422.114 23.5572 423.912 21.8251 426.221 20.5459C428.529 19.2668 431.187 18.6272 434.193 18.6272C439.133 18.6272 442.837 20.0663 445.307 22.9443C447.83 25.8223 449.092 29.8196 449.092 34.9361V58.8397H441.522V36.6149C441.522 29.2067 438.381 25.5025 432.1 25.5025C428.556 25.5025 425.791 26.6484 423.805 28.9402C421.819 31.1786 420.825 34.03 420.825 37.4943V58.7598Z" fill="white"/>
+<path fill-rule="evenodd" clip-rule="evenodd" d="M15.3489 1.59204C6.87194 1.59204 0 8.46398 0 16.9409V57.8714C0 66.3483 6.87194 73.2203 15.3489 73.2203H56.2793C64.7563 73.2203 71.6282 66.3483 71.6282 57.8714V16.9409C71.6282 8.46398 64.7563 1.59204 56.2793 1.59204H15.3489ZM30.1219 22.0572C19.1373 22.0572 10.2326 30.962 10.2326 41.9465V52.7551H20.4652V32.2898H30.6978V52.7551H40.9304V32.2898H43.4886C47.727 32.2898 51.163 35.7258 51.163 39.9643V52.7551H61.3956V41.9465C61.3956 30.962 52.4909 22.0572 41.5063 22.0572H40.9304V32.2898H30.6978V22.0572H30.1219Z" fill="white"/>
+</svg>
```

Comment:
Ran this through [SVGOMG](https://jakearchibald.github.io/svgomg/) and reduces the file size by about 50%:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="450" height="74" fill="#fff">
  <path d="M112 58.8h-7.6v-39h6.7l.8 5c1-1.9 2.5-3.4 4.5-4.5a15 15 0 0 1 7.2-1.7c6.1 0 10.3 2.7 12.5 8 1-2.5 2.5-4.4 4.7-5.8 2.3-1.5 5-2.2 8-2.2 4.5 0 8 1.3 10.7 4 2.5 2.6 3.8 6.1 3.8 10.7v25.5H156V35.1c0-3-.8-5.4-2.4-7.1a8.2 8.2 0 0 0-6.4-2.6c-2.8 0-5.1 1-6.9 2.8a10.5 10.5 0 0 0-2.6 7.5v23.1h-7.5V35c0-3-.7-5.4-2.3-7a8.3 8.3 0 0 0-6.4-2.5 9 9 0 0 0-6.9 2.8 10.3 10.3 0 0 0-2.6 7.4v23.1Zm71 1c-4.3 0-7.6-1-10-3.2a11 11 0 0 1-3.5-8.6c0-3.6 1.2-6.5 3.7-8.6 2.6-2.2 6.2-3.5 11-3.8l12.3-1v-1.1c0-6-3-8.9-9.2-8.9-3.2 0-5.7.7-7.5 2a6.6 6.6 0 0 0-2.5 5.5h-6.6c0-4 1.5-7.3 4.6-9.7a19 19 0 0 1 12.3-3.8c5 0 9 1.4 11.9 4 3 2.7 4.4 6.6 4.4 11.7v24.5h-6.4l-.8-6.3c-1 2.3-2.8 4-5.3 5.4a18 18 0 0 1-8.5 1.9Zm2-5.8c3.7 0 6.5-1.1 8.5-3.2 2-2.2 3.1-5.2 3.1-8.8v-2.1l-10 .8c-6.3.6-9.5 3-9.5 7 0 2 .7 3.6 2.1 4.7a10 10 0 0 0 5.9 1.6Zm34.6 4.8H212v-39h6.8l.7 5c1-1.9 2.5-3.4 4.6-4.5a15 15 0 0 1 7.1-1.7c6.2 0 10.3 2.7 12.5 8 1-2.5 2.5-4.4 4.8-5.8a14 14 0 0 1 8-2.2c4.5 0 8 1.3 10.6 4 2.6 2.6 3.8 6.1 3.8 10.7v25.5h-7.4V35.1c0-3-.8-5.4-2.4-7.1a8.2 8.2 0 0 0-6.3-2.6c-2.9 0-5.2 1-7 2.8a10.5 10.5 0 0 0-2.5 7.5v23.1h-7.5V35c0-3-.8-5.4-2.4-7a8.3 8.3 0 0 0-6.3-2.5 9 9 0 0 0-7 2.8 10.3 10.3 0 0 0-2.5 7.4v23.1Zm67 0H279v-39h6.8l.7 5a11 11 0 0 1 4.6-4.5 15 15 0 0 1 7.2-1.7c6 0 10.3 2.7 12.5 8 .9-2.5 2.4-4.4 4.7-5.8 2.3-1.5 5-2.2 8-2.2 4.5 0 8 1.3 10.6 4 2.6 2.6 3.9 6.1 3.9 10.7v25.5h-7.4V35.1c0-3-.8-5.4-2.5-7.1a8.2 8.2 0 0 0-6.3-2.6c-2.9 0-5.2 1-7 2.8a10.5 10.5 0 0 0-2.5 7.5v23.1h-7.5V35c0-3-.8-5.4-2.3-7a8.3 8.3 0 0 0-6.4-2.5 9 9 0 0 0-7 2.8 10.3 10.3 0 0 0-2.5 7.4v23.1Zm62.7-4.8a20 20 0 0 1-5.8-14.7c0-6 1.9-11 5.8-14.8a20 20 0 0 1 14.7-5.8 20 20 0 0 1 20.5 20.5c0 6-2 11-5.8 14.8a20 20 0 0 1-14.7 5.8c-6 0-10.9-2-14.7-5.8Zm5.4-24.7a14.1 14.1 0 0 0-3.6 10c0 4 1.2 7.4 3.6 10 2.4 2.5 5.5 3.8 9.3 3.8a12 12 0 0 0 9.3-3.9c2.4-2.6 3.6-5.9 3.6-10 0-4-1.2-7.3-3.6-9.9a12 12 0 0 0-9.3-3.9 12 12 0 0 0-9.3 3.9Zm46.4 29.5h-7.5V26.1h-7.8v-6.4h7.8V7.5h7.5v12.2h7.8v6.4H401v32.7Zm19.7 0h-7.5V0h7.5v25.7a15 15 0 0 1 5.4-5.2c2.3-1.2 5-1.9 8-1.9 5 0 8.6 1.5 11.1 4.3 2.5 3 3.8 7 3.8 12v24h-7.6V36.5c0-7.4-3.1-11.1-9.4-11.1-3.5 0-6.3 1.1-8.3 3.4-2 2.3-3 5.1-3 8.6v21.3Z"/>
  <path fill-rule="evenodd" d="M15.3 1.6C7 1.6 0 8.5 0 16.9v41c0 8.4 6.9 15.3 15.3 15.3h41c8.5 0 15.3-6.9 15.3-15.3v-41c0-8.4-6.8-15.3-15.3-15.3h-41ZM30.1 22c-11 0-19.9 8.9-19.9 19.8v10.9h10.3V32.3h10.2v20.5h10.2V32.3h2.6c4.2 0 7.7 3.4 7.7 7.7v12.8h10.2V41.9c0-11-9-19.8-19.9-19.8H41v10.2H30.7V22h-.6Z" />
</svg>
```